pkgs/development/tools/continuous-integration/woodpecker/server.nix

```diff
@@ -1,6 +1,21 @@
-{ buildGoModule, callPackage }:
+{
+  buildGoModule,
+  callPackage,
+  nodejs,
+  pnpm,
+}:
 let
   common = callPackage ./common.nix { };
+
+  nodeModulesHash = "sha256-34N20FvgBbJAa28u56ZrYrT16J/+8OPuIm1O7EYGVc0=";
```

Comment:
Nit: I think this belongs in `common.nix` (the idea being this is the only file modified by the update script).

Reply:
done